flink-formats/flink-avro/src/main/java/org/apache/flink/formats/avro/AvroToRowDataConverters.java

```diff
@@ -178,7 +178,7 @@ private static AvroToRowDataConverter createArrayConverter(ArrayType arrayType)
 
 	private static AvroToRowDataConverter createMapConverter(LogicalType type) {
 		final AvroToRowDataConverter keyConverter = createConverter(DataTypes.STRING().getLogicalType());
```

Comment:
key can also be `null`?

Reply:
Avro can't serialize null key for map.

Reply:
> Avro can't serialize null key for map.

That's right, and this problem only exists when the type of map entry is `STRING()`